apps/website/src/pages/join-us/[ashbyIdOrCmsSlug].tsx

```
@@ -0,0 +1,218 @@
+    <div>
+      {loading && <ProgressDots />}
+      {(error || (data && !job)) && <ErrorSection error={error || new Error('Job not found. This job posting may have been closed.')} />}
+      {job && (
+        <>
+          <Head>
+            <title>{`${job.title} | BlueDot Impact`}</title>
+            <meta name="description" content={job.location} />
+            <script
+              type="application/ld+json"
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)